dvc/output/__init__.py

```diff
@@ -25,10 +27,17 @@
             's3': OutputS3,
             'gs': OutputGS,
             'ssh': OutputSSH,
-            '': OutputLOCAL}
+            'local': OutputLOCAL}
 
-SCHEMA[schema.Optional(OutputLOCAL.PARAM_CACHE)] = bool
-SCHEMA[schema.Optional(OutputLOCAL.PARAM_METRIC)] = OutputLOCAL.METRIC_SCHEMA
+# We are skipping RemoteHTTP.PARAM_CHECKSUM because is the same as RemoteS3
```

Comment:
Thinking about it, it's not that straight forward that we have three different checksums here: `md5`, `etag`, `checksum`; Maybe we could improve this comment to clarify that